counter-script/project/scripts/interaction_script/src/main.sw

```diff
@@ -2,6 +2,16 @@ script;
 
 use libraries::Counter;
 
+/// Calls the methods of the counter contract at the given contract_id.
```

Comment:
```suggestion
/// Calls the functions of the counter contract at the given contract_id.
```

Reply:
Why?

Reply:
https://fuellabs.github.io/sway/master/reference/documentation/language/functions/index.html

Reply:
https://fuellabs.github.io/sway/master/reference/documentation/examples/counter.html?highlight=Contract%20method#implementation

Reply:
![d84b3210af89d64439c07e7c7b3c073367b834c65c4ed76f4c72a77c87a011d9_1-457759858](https://github.com/FuelLabs/sway-applications/assets/103777923/78db4f37-185a-47e2-a33f-75b969d3df69)

Yh, that should be fixed too

Reply:
No, the term methods is correct. Even in the rust sdk you do .methods on the contract object to access the methods remember?

Reply:
It's an associated function. For UX you're going to call it a `.method()` in the SDK.

Reply:
Moving ahead with method, as per internal discussion